### ISSUE TITLE
contentScript sourceMap does not work again

### ISSUE THREAD

**Issue by senkenn** [wxt-dev/wxt#912]
### Describe the bug

contentScript sourceMap does not work again. 
Ref: #236 

### Reproduction

![image](https://github.com/user-attachments/assets/caacd94e-f1fe-444f-b207-dfcc3adb0237)

### Steps to reproduce

1. pnpm dlx wxt@latest init foo-example
2. pnpm i
3. pnpm dev
4. access to google.com
5. open developer tools
6. click console tab

then you can check content script output.

### System Info

```shell
System:
    OS: Linux 6.5 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
    CPU: (16) x64 AMD Ryzen 7 5700U with Radeon Graphics
    Memory: 5.42 GB / 14.94 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 20.14.0 - ~/.nvm/versions/node/v20.14.0/bin/node
    npm: 10.7.0 - ~/.nvm/versions/node/v20.14.0/bin/npm
    pnpm: 9.7.0 - ~/.nvm/versions/node/v20.14.0/bin/pnpm
    bun: 1.0.35 - ~/.bun/bin/bun
  Browsers:
    Brave Browser: 127.1.68.134
    Chrome: 127.0.6533.99
  npmPackages:
    wxt: ^0.19.1 => 0.19.4 
```

### Used Package Manager

pnpm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by aklinker1** [comment]
TLDR: Setting changed from camel case to kebab case, will update it. 

---

For my future reference:

This is caused by content scripts not being allowed to load source maps by default.

<img width="369" alt="Screenshot 2024-08-12 at 10 07 56 AM" src="https://github.com/user-attachments/assets/ce29fe0c-9a22-4bee-be5a-c292e521d701">

You can enable them via devtool settings:

<img width="571" alt="Screenshot 2024-08-12 at 10 08 58 AM" src="https://github.com/user-attachments/assets/753eb944-cb26-4216-b8bd-569ab171a816">

I already added this preference to the default preferences:
https://github.com/wxt-dev/wxt/blob/72673ca366b565730e62b7164e57404b25d7acbf/packages/wxt/src/core/runners/web-ext.ts#L93-L102

But yeah, you're right. Looks like google renamed the setting:
- https://source.chromium.org/search?q=%22Content%20scripts%20injected%20by%20extensions%22&ss=chromium
- https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/panels/settings/FrameworkIgnoreListSettingsTab.ts;l=116;drc=0cb974616f47fbf4fb51e9f57383ef39eb4fb58e

**Comment by aklinker1** [comment]
Fixed by https://github.com/wxt-dev/wxt/commit/e6529e6e41bb8bda28068328c0d8859a958f441e

**Comment by aklinker1** [comment]
Released in [`v0.19.5`](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.5)

**Comment by senkenn** [comment]
@aklinker1 Thank you! I also could confirm it worked!

**Comment by marijnbent** [comment]
Ah, I get the following error when going to the content.js script in DevTools. I'm not running adblockers or something like that. Does someone know whats causing this?

<img width="332" alt="Image" src="https://github.com/user-attachments/assets/862a2940-403d-4716-a174-2be17b4137c6" />

**Comment by marijnbent** [comment]
@aklinker1 With a fresh wxt install (Vue), and only adding the snippet below with the sourcemap value as true, the sourcemap still does not load for me if I click on the log on google.com (default content script). Can you try this on your end to validate?

For others looking for a solution, set your sourcemaps to 'inline' for now!

``` wxt.config.js
export default defineConfig({
    ...
    vite: () => ({
        build: {
            sourcemap: 'inline',
        },
    }),
```

**Comment by aklinker1** [comment]
This is intended, see https://github.com/wxt-dev/wxt/blob/main/packages%2Fmodule-vue%2Fsrc%2Findex.ts#L16-L17

**Comment by marijnbent** [comment]
I don't get the error when the value is set to true, but the sourcemap still doesn't load in Chrome though. Everything is build correctly

**Comment by spookyuser** [comment]
Same here w/ react  setting source map to inline does work though but the page feels much slower with this on

 ```
build: {
      sourcemap: mode === "development" ? "inline" : false,
}
```

**Comment by northsea4** [comment]
Any updates on the Vue issue? Developing without source maps is terrible!
